server/handler/base.go

```diff
@@ -91,6 +92,14 @@ func (p *PullEvaluationOptions) FillDefaults() {
 	}
 }
 
+func (p *PullEvaluationOptions) SetValuesFromEnv(prefix string) {
+	setStringFromEnv("POLICY_PATH", prefix, &p.PolicyPath)
+	setStringFromEnv("SHARED_REPOSITORY", prefix, &p.SharedRepository)
+	setStringFromEnv("SHARED_POLICY_PATH", prefix, &p.SharedPolicyPath)
+	setStringFromEnv("STATUS_CHECK_CONTEXT", prefix, &p.StatusCheckContext)
+	p.fillDefaults()
+}
```

Comment:
i think maybe this is the better way mirroring your pattern from go-baseapp and go-githubapp, but i felt like these options weren't suited to live on in those codebases. so i put them here instead. 

only thing i felt weird about was setting the defaults here, probably fine, but let me know what you think

Reply:
I agree it's a bit weird, but I think it's fine since this is only used in one place.